src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
```

Comment:
If you remove the unused line of code inside of your CSS ruleset, please do not forget to remove the empty line as well. It will help  to keep your code clean.